src/Middleware/integrations/OrderCloud.Integrations.EasyPost/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Headstart.Common.Commands;
+using Headstart.Common.Services;
+using Headstart.Common.Settings;
+using Microsoft.Extensions.DependencyInjection;
+using OrderCloud.Integrations.EasyPost.Commands;
+
+namespace OrderCloud.Integrations.EasyPost.Extensions
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IServiceCollection AddEasyPostShippingProvider(this IServiceCollection services, EnvironmentSettings environmentSettings, EasyPostSettings easyPostSettings)
+        {
+            if (!environmentSettings.ShippingProvider.Equals("EasyPost", StringComparison.OrdinalIgnoreCase))
+            {
+                return services;
+            }
```

Comment:
We should  add a null conditional operator here otherwise we get an error. Same for the rest 

![image](https://user-images.githubusercontent.com/16483662/173857795-2d648ae7-890b-4ecc-834b-4f069cc457fa.png)